packages/ds-medicare-gov/src/styles/components/_override.help_drawer.scss

```diff
@@ -35,22 +35,48 @@
 /* stylelint-enable */
 
 // Help drawer button with icon
+// TODO: clean up this code if a different, more flexible SVG can be used instead
 .ds-c-help-drawer__toggle {
-  background: url('#{$image-path}/InfoOutline.svg') center right no-repeat;
-  padding-right: $spacer-3;
+  .icon-wrapper {
+    position: relative;
+  }
+
+  .ds-c-icon--info-circle-outline {
+    width: 18px;
+    height: 18px;
```

Comment:
Did the original one change size on hover as well? It looks like this one shrinks

Reply:
I see that it's impractical to fix this right now.